study.md

```diff
@@ -43,5 +43,6 @@ your deployed Ember app? What do you need if you want to use the `history` API
 instead of `hash` for `location`?
 
 ```md
-<!-- your answer here -->
+Hash URLs are URLs that includes a hash. The hash
+allows users to navigate from page to page without requiring a page refresh. It uses the URL's anchor to load the starting state of your all and keep it in sync as users move around the app.
```

Comment:
Yes. This is important with GitHub Pages because GitHub Pages doesn't have a persistent Ember server running to serve up new pages - it just serves up the one static page, which runs Ember as part of its JS code.

Incidentally, as a result of this, a page refresh will cause the entire Ember app to reset when using GH Pages.